content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -35,7 +35,7 @@ a container that writes some text to standard output once per second.
 To run this pod, use the following command:
 
 ```shell
-$ kubectl create -f https://k8s.io/examples/debug/counter-pod.yaml
+$ kubectl apply -f https://k8s.io/examples/debug/counter-pod.yaml
 pod/counter created
```

Comment:
Hello @Liujingfang1, in general, is `kubectl create` being phased out in favor of `kubectl apply`?
I'll look a bit further for an explanation on the change and a description of the combined action.

Reply:
To be precise, we recommend to use `kubectl apply -f` over `kubectl create -f`. 
`kubectl create` accepts other flags and arguments, we don't change docs for them.